### ISSUE TITLE
can't download package vary@dc17535 as the url changed

### ISSUE THREAD

**Issue by youngyou** [asos-craigmorten/opine-http-proxy#1]
# Issue

Setup:

- Deno Version: 1.2.0
- v8 Version: 8.5.216
- Typescript Version: 3.9.2
- Opine HTTP Proxy Version: 2.0.0
- Opine Version: 0.20.1

Can't download package vary@dc17535 as the url changed

## Details

When I bundle the code,  it shows:
```bash
error: Import 'https://deno.land/x/vary@dc17535/mod.ts' failed: 404 Not Found
Imported from "https://deno.land/x/opine@0.19.1/deps.ts:46"
```
Ref: https://github.com/asos-craigmorten/opine/issues/48

**Comment by asos-craigmorten** [comment]
Heya @youngyou 👋 

Being slow this morning and thought was looking at issues in a different repo 🤦🏻‍♂️ (in case saw previous messages which have deleted)

Upgrading this repos deps should do the trick, will get a fix out ASAP!

**Comment by asos-craigmorten** [comment]
Ah are you using the proxy module? Don’t think I’ve touched that in a while and would certainly explain why your code is still importing 0.19.1 if you are! If so it’s a bug with that and we should get an issue raised there and a fix in ASAP 😄

**Comment by asos-craigmorten** [comment]
Hopefully fixed in `2.1.0` - let me know if have further issues.
